lib/api/dsl/methods.js

```diff
@@ -690,15 +760,17 @@ module.exports = methods = {
     }
 
     try {
-      /*eslint guard-for-in: 0 */
       for (i in Object.keys(filter)) {
```

Comment:
Can we do something like:

```
Object.keys(foo).forEach(function (bar) {
  ...
});
```

In order to standardize with the rest of code ?

And for avoid lines
```
if (!Object.keys(filter).hasOwnProperty(i)) {
   continue;
}
```